perfkitbenchmarker/linux_benchmarks/sysbench_benchmark.py

```diff
@@ -356,8 +352,6 @@ def Prepare(benchmark_spec):
   prepare_results = _PrepareSysbench(vm, metadata, benchmark_spec)
   print prepare_results
```

Comment:
Are these just lost?  We probably should not be printing them regardless.

Reply:
The assignment to the global was not doing anything. Printing the data could still be useful. Nathan, any comment?